### ISSUE TITLE
I can't upload designs from sketch

### ISSUE THREAD

**Issue by leonenko** [avocode/avocode#1526]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.6`
Avocode UID: 69739
```
Error: Api#updateUploadSlot()() -> error (status: 404, error: "Not found")
    at Object.normalizeError (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16898)
    at Object.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16734)
    at t.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1994821)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1991839
    at Request._callback (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1993662)
    at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:186:22)
    at emitTwo (events.js:111:20)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1001:12)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by jakubzitny** [comment]
Hi @leonenko, thanks for reporting this problem, I am sorry to hear that it doesn't work for you. Does this happen for all Sketch uploads or just one/some? When exactly does the error appear, please? Can you reproduce it?

**Comment by bolekkerous** [comment]
Hey, @leonenko  -
could you please try to log out and log in again? It seemed to help in a similar issue here: #1611

**Comment by jakubzitny** [comment]
Hi there, this should be fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
